compiler/rustc_codegen_llvm/src/gotoc/mod.rs

```diff
@@ -97,13 +97,43 @@ impl<'tcx> GotocCtx<'tcx> {
         self.current_fn().mir().basic_blocks().indices().map(|bb| format!("{:?}", bb)).collect()
     }
 
+    fn should_skip_current_fn(&self) -> bool {
+        match self.current_fn().readable_name().as_str() {
+            // https://github.com/model-checking/rmc/issues/202
+            "fmt::ArgumentV1::<'a>::as_usize" => true,
+            // https://github.com/model-checking/rmc/issues/203
+            "<(dyn core::any::Any + core::marker::Send + 'static)>::downcast_ref" => true,
+            // https://github.com/model-checking/rmc/issues/204
+            "sys_common::thread_info::THREAD_INFO::__getit" => true,
+            // https://github.com/model-checking/rmc/issues/205
+            "panic::Location::<'a>::caller" => true,
+            // https://github.com/model-checking/rmc/issues/206
+            "core::sync::atomic::atomic_swap" => true,
+            // https://github.com/model-checking/rmc/issues/207
+            "core::slice::<impl [T]>::split_first" => true,
+            // https://github.com/model-checking/rmc/issues/208
+            "panicking::take_hook" => true,
+            // https://github.com/model-checking/rmc/issues/209
+            "panicking::r#try" => true,
+            _ => false,
+        }
+    }
+
```

Comment:
It might be nice in the future to read these from a single file, that the README can then point to to document what's skipped.

Reply:
Good idea